static/js/add-shows/new-show.js

```diff
@@ -41,8 +41,10 @@ MEDUSA.addShows.newShow = function() {
         sampleText += '</b>';
 
         // if we have a show name then sanitize and use it for the dir name
-        if (showName.length) {
-            $.get('addShows/sanitizeFileName', {name: showName}, function (data) {
+        if (showName.length > 0) {
```

Comment:
I think it's more what we agree on as being a standard. opposed to using 5 different ways of checking the length of an array.